src/components/Currencies.js

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import store from '../redux/configureStore';
+import { combineCurrencyRate, filterRates } from '../utils/utils';
+
+import '../styles/Currencies.css';
+
+const Currencies = () => {
+  const currencies = useSelector((state) => state.currencies);
+  const usdExchange = useSelector((state) => state.usdExchange);
+
+  const [rates, setRates] = useState([]);
```

Comment:
- [ ] Please write API and unit tests for components.

Reply:
Done.